src/components/Auth/SignUp.tsx

```diff
@@ -87,16 +135,42 @@ const SignUp = () => {
             type="password"
             name="password"
             id="password"
+            value={password}
+            onChange={(e) => setPassword(e.target.value)}
           />
 
-          <Button className={styles.cardButton}>Sign in</Button>
+          <Button type="submit" className={styles.cardButton} loading={loading}>
+            Sign Up
+          </Button>
         </form>
 
+        {error && (
+          <Notification
+            icon={<IconX size={18} />}
+            color="red"
+            title="Bummer!"
```

Comment:
we can abstract this to const at the top of file - ERROR_TITLE, and SUCCESS_TITLE